actioncable/app/assets/javascripts/action_cable/connection.coffee

```diff
@@ -92,8 +95,15 @@ class ActionCable.Connection
       @disconnected = false
       @subscriptions.reload()
 
-    close: ->
+    close: (event) ->
       ActionCable.log("WebSocket onclose event")
+      if event.code == 1006
+        ActionCable.log("WebSocket received abnormal close error (" + event.code + "): " + event.reason)
+      
+      if !event.wasClean
```

Comment:
In CoffeeScript you can do `unless event.wasClean` (or `if not event.wasClean`).